tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -142,10 +147,11 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
 
   test("cuDF does not support quantifier syntax when not quantifying anything") {
     // note that we could choose to transpile and escape the '{' and '}' characters
-    val patterns = Seq("{1,2}", "{1,}", "{1}", "{2,1}")
```

Comment:
My guess here is that cuDF is now smart enough to realize that `{2,1}` is not a qualifier but need to confirm and a test in the follow on PR